src/plone/folder/tests/test_partialordering.py

```diff
@@ -67,7 +67,11 @@ def runTableTests(self, action, tests):
             container, ordering = self.create()
             ids = set(container.objectIds())
             method = getattr(ordering, action)
-            if isinstance(rval, Exception):
+            if isinstance(rval, type):
```

Comment:
@mauritsvanrees I suspect that what's actually intended here is `issubclass(rval, Exception)`

Reply:
I did not know the `issubclass` function. Thanks!

It works for this case, but gives an error when you pass an integer like in the other tests:

```
TypeError: issubclass() arg 1 must be a class
```

Ah, but we can combine them:

```
if isinstance(rval, type) and issubclass(rval, Exception):
```

Then it works and is clear.

Done.

Reply:
@mauritsvanrees Well that's annoying! Obviously it's not a subclass if it's not a class! But, good enough.